gadopt/approximations.py

```diff
@@ -3,7 +3,7 @@
 from .utility import ensure_constant, vertical_component
 
 
-class BaseApproximation:
+class BaseApproximation(metaclass=abc.ABCMeta):
```

Comment:
I think inheriting `abc.ABC` directly would be a little cleaner and more idiomatic, i.e.

```python
class BaseApproximation(abc.ABC):
```

Reply:
I saw that argument, yes, but then I also found that Python developers seem to use the `metaclass` keyword (at least [here](https://github.com/python/cpython/blob/67e8d416cc5e14c181e4f883ffb4d87f2a647ae1/Lib/_collections_abc.py#L131)). Up to you honestly. :)

Reply:
I think it's a bit of both (e.g. [PathLike](https://github.com/python/cpython/blob/67e8d416cc5e14c181e4f883ffb4d87f2a647ae1/Lib/os.py#L1084)). Consistency be damned!